core/src/main/scala/codesearch/core/regex/lexer/Tokenizer.scala

```diff
@@ -13,6 +13,9 @@ object Tokenizer {
   /** A POSIX character class, e.g. `[:alpha:]`. */
   private def charSetPred[_: P] = P("[:" ~ (!":]" ~ AnyChar).rep ~ ":]")
 
+  private def parseRepetitionSeq[_: P] =
```

Comment:
You should add a comment specifying which cases are parsed (`{n,m}`, `{n,}` and `{n}`).